src/components/modules/toolbar/index.ts

```diff
@@ -381,8 +381,6 @@ export default class Toolbar extends Module<ToolbarNodes> {
    * It is used in Read-Only mode
    */
   private destroy(): void {
-    this.Editor.Toolbox.destroy();
```

Comment:
This destroy is also called when we [toggle read only mode](https://github.com/codex-team/editor.js/blob/75d6592cd7510e68cf99634992412ced764de260/src/components/modules/toolbar/index.ts#L152). Would that expect this destroy to take care of destroying toolbox and blockSettings? Just curious as we use this in production.

https://github.com/codex-team/editor.js/blob/75d6592cd7510e68cf99634992412ced764de260/src/components/modules/toolbar/index.ts#L152

Reply:
@sis-dk 
Ya that would be handle with toggle button. We just removing this because it called twice when we trying to destroy the editor.

Reply:
@sis-dk @robonetphy thanks. I've moved the Toolbox and BlockSettinga destroy methods to the `toggleReadOnly`, because it is the logical opposite to `this.drawUI()`.

1. On the read-only mode toggling all the modules destroys correctly, no memory leak here. 
2. On `editor.destroy()` all modules` destroy fired once.